src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -364,21 +364,22 @@ void Rules::sendWarningsForDisabledClusters()
         int nbScenariiForCluster = (int)scenariiForCurrentCluster.size();
         vector<uint>::iterator itv = scenariiForCurrentCluster.begin();
 
-        // Listing the 10 first scenarii numbers (years numbers) where current cluster is refered
-        // to. Notice that these scenarii could be less then 10, but are at least 1.
-        string listScenarii = to_string(*itv + 1);
+        // Listing the 10 first years for which the current cluster was given a specific TS number 
+        // in the scenario builder.
+        // Note that this list of years size could be less then 10, but are at least 1.
+        string listYears = to_string(*itv);
         itv++;
-        for (; itv != scenariiForCurrentCluster.end(); itv++)
-            listScenarii += ", " + to_string(*itv + 1);
+        for (int year_count = 1; itv != scenariiForCurrentCluster.end() && year_count < 10; itv++, year_count++)
```

Comment:
Starting at 1, you'll only get 9 years. Start at 0 for 10 ?

Reply:
Before the loop **for**, **listYears** is filled with 1 year.
In the loop, **listYears** is filled with 9 more years.